src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,29 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+  const isComments = comments.length !== 0;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <UserInfo user={user} />
```

Comment:
user can be undefined, so let's add the check here, and render this component only when a user exists
```suggestion
       {user && <UserInfo user={user} />}
```